pkg/controller/certificates/sync.go

```diff
@@ -57,8 +58,14 @@ const (
 	messageIssuingCertificate  = "Issuing certificate..."
 	messageRenewingCertificate = "Renewing certificate..."
 
+	infoCertificateDuration = "WarnCertificateDuration"
+	infoScheduleModified    = "WarnScheduleModified"
+
 	messageCertificateIssued  = "Certificate issued successfully"
 	messageCertificateRenewed = "Certificate renewed successfully"
+
+	messageCertificateDuration = "Certificate received from server has a validity duration of %s. The requested certificate validity duration was %s"
+	messageScheduleModified    = "Certificate renewal duration was changed to fit inside the received certificate validity duration from issuer."
```

Comment:
These are only used in one place, and for glog messages.

Can you move them down into the function body instead of declaring here? I am (hoping to) gradually move away from having these random string format definitions at the top of files, as it makes it difficult to read the code.